src/tests/ftest/aggregation/multiple_pool_cont.py

```diff
@@ -0,0 +1,195 @@
+
+
+class DaosAggregationMultiPoolCont(IorTestBase):
+    # pylint: disable=too-many-ancestors
+    """Test class Description:
+
+       Run IOR with same file option to verify the
+       aggregation.
+
+    :avocado: recursive
+    """
+    def setUp(self):
+        super().setUp()
+        self.free_space_dict = {}
+
+    def save_free_space(self, space_tag, storage_index):
+        """Save the storage space in free_space dictionary for each pool
+        self.free_space_dict[pool][0] = val [Initial free space]
+        self.free_space_dict[pool][1] = val [After 1st IOR free space]
+        self.free_space_dict[pool][2] = val [After 2nd IOR free space]
+        self.free_space_dict[pool][3] = val [After aggregation complete]
+
+        Args:
+            storage_index (int): index of the pool free space to obtain
+        """
+        for pool in self.pool:
+            pool.get_info()
+            value = pool.info.pi_space.ps_space.s_free[storage_index]
+            if space_tag == 0:
+                self.free_space_dict[pool.uuid] = {space_tag: value}
+            else:
+                self.free_space_dict[pool.uuid][space_tag] = value
+            self.log.info("Free Space Information")
+            self.log.info("======================")
+            self.log.info(self.free_space_dict)
+
+    def test_free_space(self, space_tag1, space_tag2):
+        """Test whether the free space is decremented between two IOR runs.
+           Asserts if free sapce with space_tag1 less than free space with
+           space_tag2.
+
+        Args:
+            space_tag1 : IOR run tag (0,1,2)
+            space_tag2 : IOR run (1,2,3)
+        """
+        for pool in self.pool:
+            space_used_by_ior = (self.free_space_dict[pool.uuid][space_tag1] -
+                                 self.free_space_dict[pool.uuid][space_tag2])
+            self.log.info("Pool %s Space used by ior = %s", pool.uuid,
+                          space_used_by_ior)
+            self.assertTrue(self.free_space_dict[pool.uuid][space_tag1] >
+                            self.free_space_dict[pool.uuid][space_tag2],
+                            "IOR run was not successful.")
+
+    def test_longrun_aggregation(self, total_pools=1, total_containers_per_pool=1):
+        """Jira ID: DAOS-7326
+        Test Description:
+            This is a common method which run IOR on the specified pools,
+            container quanties and test aggregation.
+        """
+        # test params
+        total_runtime = self.params.get("total_runtime",
+                                        '/run/runtime/*')
+        pool_qty = total_pools
+        container_qty = total_containers_per_pool
+        start_time = 0
+        finish_time = 0
+
+        job_manager = get_job_manager(self, "Mpirun", None, False, "mpich",
+                                      self.get_remaining_time())
+        # Create requested pools
+        self.add_pool_qty(pool_qty, connect=False)
```

Comment:
```suggestion
        self.add_pool_qty(total_pools, connect=False)
```
Can't we use `total_pools` directly here, then remove line 76?

Reply:
Will do.

Reply:
Done.